src/kubectl-asset.ts

```diff
@@ -1,26 +1,16 @@
-import * as crypto from 'crypto';
-import * as fs from 'fs';
-import * as path from 'path';
 import * as s3_assets from 'aws-cdk-lib/aws-s3-assets';
 import { Construct } from 'constructs';
+import { assetHash, ASSET_FILE } from './_asset';
```

Comment:
Why not just remove this file?

Reply:
We could. It's breaking, but we don't care?

Reply:
Well, we are also gonna change the name of the package! So it's not breaking.